book/src/external-snapshotter.md

```diff
@@ -4,34 +4,61 @@
 
 **Git Repository:** [https://github.com/kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter)
 
-**Status:** Alpha
+**Status:** Beta
```

Comment:
Should we also have information on which versions were Beta and which were Alpha?

Reply:
Sure